index.html

```diff
@@ -147,40 +169,57 @@ <h2>Uploading New Materials</h2>
 				<p style="padding-bottom: 20px;">Your files should automatically upload. There is no limit to the file sizes you can upload however, please be aware that the larger the file, the longer it will take to upload. Depending on how large the file, the server may timeout the request (these are for files cumulating larger than a few GB). To prevent this, it is advised you upload portions of your object at a time depending on file size. You can upload exercise solution files, but they will be identified and not distributed through CLARK. A content consumer will be provided your email to obtain solutions directly from you.</p>
 				<div class="picture">
 					<img src="./Resources/CLARK-Materials-Upload.png" width="1000px">
-					<p style="padding-top: 20px;">You have now successfully created your first learning object!</p>
 				</div>
 			</div>
 			<br><br>
-		
+
+			<a name="Children"><div class="section">
+				<h2>Adding Children Learning Objects</h2>
+				<hr>
+				<p>When you upload a large learning object that uses many smaller objects in it, it is useful to add these objects as <i>children</i> to the larger one.  This is to help organize your larger learning objects and allow others to easily find all the parts of your curriculum.</p>
```

Comment:
"When you upload a large learning object that uses many smaller objects in it"
Should be:
"When you upload a large learning object (such as a course or unit) that is made up of many smaller objects (such as a nano-module or micro-module) in it"

Additionally, "This is to help organize your larger learning objects and allow others to easily find all the parts of your curriculum."
Should be:
"This is to help organize your larger learning objects and allow others to selectively download parts of your object. For instance, if a consumer downloads a course you created but accidentally lost the files to a lab they were planning on covering, they won't need to download the entire course again, only the specific object they are missing."